### PR TITLE
WRN-12940: Handle 5-way nav with color pickers.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Created a new HSL ColorPicker to work with 5 way. Fixed a bug for TipleField. 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-12940

### Comments
Enact-DCO-1.0-Signed-off-by: Alexandru Morariu alexandru.morariu@lgepartner.com

### CODE REVIEW

File: sandstone/custom-skin/src/components/ColorPicker/ColorPicker.module.less [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.colorPickerPanel {
+  --sand-focus-bg-color: #fff;
+  --sand-overlay-bg-color-rgb: 76, 89, 106;
+  --sand-text-color: #e6e6e6;
+  width: 840px;
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/ColorPicker/ColorPicker.module.less:5]: Could you change px to rem?

**Review by stanca-pop-lgp — COMMENTED** [review]
Please check out my comments.

File: sandstone/custom-skin/src/components/ColorPicker/ColorPicker.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+				className={css.colorBlock}
+				disabled={disabled}
+				onClick={openPopup}
+				style={{
+					backgroundColor: color,
+					'--sand-focus-bg-color': color
+				}}
+				type="color"
+			/>
+			<Popup
```
[the remark this answers is not in the document]
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/components/ColorPicker/ColorPicker.js:55]: ContextualPopup has some problems with keeping spotlight on keypress. Will investigate further how to properly use it.

[the remark this answers is not in the document]
Comment (stanca-pop-lgp) [reply sandstone/custom-skin/src/components/ColorPicker/ColorPicker.js:55]: I agree with keeping the popup for now (for the coming demo) and switching to contextualPopup if/when everything works well. 
Just one more suggestion. Maybe make the color button a simple div (so it doesn't focus) and center it in the popup.
image

[the remark this answers is not in the document]
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/components/ColorPicker/ColorPicker.js:55]: Done

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**